lang/cs/Org.Apache.REEF.Network/Group/Operators/IGroupCommOperator.cs

```diff
@@ -39,5 +39,7 @@ public interface IGroupCommOperator<T>
         /// The operator version number.
         /// </summary>
         int Version { get; }
+
+        void Initialize();
```

Comment:
Please add documentation as to what this method does and when it needs to be called. The name to me suggests something that could (potentially) be done in a constructor?